### PR TITLE
Implement sample test using Playwright

### LINKED ISSUE #4: Setup Microsoft Playwright
## TODO

- [ ] Install Playwright and related dependencies

### LINKED ISSUE #5: Implement sample test using Playwright
Implement a simple test in Playwright that opens Wikipedia

## TODO

- [ ] Implement the simplest possible test that opens Wikipedia homepage

### PR DESCRIPTION
**Pull request by hariclerry**
### Tickets
Closes #5 

### What does this PR do?
This PR implements a sample test using Playwright

### What are the changes made?
- Created a playwright folder inside the tests directory
- Added wikipedia.js with playwright test
- Added a README.md file in the playwright folder with instructions on how to run the test.

### To run test
- Clone the repository and checkout to the `issues-05` branch.
- Run `npm ci` to install dependencies
- Run `node tests/playwright/specs/wikipedia.js` from the app root directory

### Screen shots
None

### CODE REVIEW

File: tests/playwright/specs/wikipedia.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var { chromium } = require("playwright");
+
+(async () => {
```
Comment (sovoid) [inline tests/playwright/specs/wikipedia.js]: I wouldnt call this a test. You need to setup a test runner like jest to run the tests. I guess I wasnt explicit. Ill update the issue

**Review by sovoid — CHANGES_REQUESTED** [review]

File: tests/playwright/specs/wikipedia.js (hunk 1 above)
Comment (sovoid) [reply tests/playwright/specs/wikipedia.js]: Write assertion for an element on the page
https://playwright.dev/#version=v1.6.2&path=docs%2Fassertions.md&q=

**Review by sovoid — CHANGES_REQUESTED** [review]

File: tests/playwright/specs/wikipedia.js (hunk 1 above)
Comment (hariclerry) [reply tests/playwright/specs/wikipedia.js]: This has been noted.

**Comment by sovoid** [comment]
@hariclerry pulls on upstream should be done in separate pull requests. Notice how the commit history of this PR gets messed up making it harder to review. I am closing this PR. Please make another PR. You can directly pull upstream on the master

**Pull request closed without merging**